db/db_impl/db_impl_readonly.h

```diff
@@ -27,6 +27,9 @@ class DBImplReadOnly : public DBImpl {
   virtual Status Get(const ReadOptions& options,
                      ColumnFamilyHandle* column_family, const Slice& key,
                      PinnableSlice* value) override;
+  virtual Status Get(const ReadOptions& options,
```

Comment:
Nit: for new code, we can keep `override` and remove `virtual`, according to https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-override.html.

Reply:
Thanks for the pointer.